#! /bin/sh /usr/share/dpatch/dpatch-run
## 17-fix_tail_syntax_in_xversion.h.dpatch by Sven Mueller <debian@incase.de>
## Updated to 2.3.13 by Duncan Gibb <duncan.gibb@siriusit.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix tail syntax in xversion.h (use "tail -n 1" instead of "tail -1")

@DPATCH@
diff -urNad git~/imap/xversion.sh git/imap/xversion.sh
--- git~/imap/xversion.sh	2010-01-16 19:21:09.000000000 -0200
+++ git/imap/xversion.sh	2010-01-16 19:29:14.090092099 -0200
@@ -62,7 +62,7 @@
 	match ($0, pattern) {
 	    printf "\"%s\"\n", substr($0, RSTART, RLENGTH)
 	}' pattern="$DATEPAT $TIMEPAT" | \
-	sort | tail -1 >> $TMPF
+	sort | tail -n 1 >> $TMPF
 
 if [ -f xversion.h ] && cmp -s $TMPF xversion.h
 then
